stories/semantic-color/semantic-color.stories.js

```diff
@@ -0,0 +1,87 @@
+export default {
+    title: 'Components/Semantic Color',
+    parameters: {
+        tags: ['f3', 'a11y', 'theme'],
+        description: `
+The semantic-color component is usefull when using semantic icons or messages.
+The default semantics are the information colors
+##Usage
+**Use notifications if:**
+    
+* You want to show a semantic icon being error, success, warning, information, question.
+* You want to show a title using a semantic color
+            `,
+        components: ['semantic-color']
+    }
+};
+
+export const information = () => `
+<div>
+    <i arial-label="information" class="fd-semantic-color__icon"></i>
+    <br>
```

Comment:
Can we avoid using `<br />` tags for styling?

Reply:
Fixed so i dont need line breaks anymore